pages/[walletId]/index.tsx

```
@@ -184,264 +562,573 @@ const Home: NextPage = () => {
+                      />
+                      <span className="text-white text-sm">NFT Mint</span>
+                    </label>
+                  </div>
+                </div>
+              )}
+              
+              <div className="grid gap-4 md:grid-cols-3">
+                <div className="md:col-span-2">
+                  <label className="block text-sm font-medium text-gray-400 mb-2">
+                    {newMemberType === 'nft' ? 'NFT Mint Address' : 'Wallet Address'}
+                  </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)